### ISSUE TITLE
transfer error not_enough_money 

### ISSUE THREAD

**Issue by mrx23dot** [monero-project/monero#7457]
I'm trying to transfer 0.000000000001 (smallest amount) from 0.000000470000 **unlocked** balance via RPC, but I get **not_enough_money** error.

I have waited more than 20mins after received the 047 to spend, and transaction has 65 confirms. 
Using lower prio. I only have one account in wallet and I'm fully synced.

So why can't I send money?

monero-wallet-rpc --wallet-file wallet --password xxx --daemon-address xmr.fail:18081 --rpc-bind-port 28089 --rpc-bind-ip 127.0.0.1 --disable-rpc-login --log-level 2
Latest release.

2021-03-06 23:23:10.536 I Refresh done, blocks received: 0, balance (all accounts): 0.000000470000, **unlocked: 0.000000470000**

2021-03-06 23:23:31.865 D Using v8 rules
2021-03-06 23:23:31.867 W Requested ring size 1 too low, using 11
2021-03-06 23:23:31.868 D Using v8 rules
2021-03-06 23:23:31.868 D on_transfer_split calling create_transactions_2
2021-03-06 23:23:32.020 D Using v5 rules
2021-03-06 23:23:32.022 D Using v8 rules
2021-03-06 23:23:32.023 D Using v8 rules
2021-03-06 23:23:32.167 D Using v4 rules
2021-03-06 23:23:32.169 D Using v8 rules
2021-03-06 23:23:32.319 D Using v13 rules
2021-03-06 23:23:32.321 D Using v13 rules
2021-03-06 23:23:32.321 D Using v4 rules
2021-03-06 23:23:32.467 D Using v8 rules
2021-03-06 23:23:32.469 D Using v8 rules
2021-03-06 23:23:32.469 D transfer: adding 0.000000000001, for a total of 0.000000000001
2021-03-06 23:23:32.469 D estimated bulletproof rct tx size for 1 inputs with ring size 11 and 2 outputs: 1416 (800 saved)
2021-03-06 23:23:32.470 E needed_money + min_fee > balance_subtotal. THROW EXCEPTION: error::not_enough_money
2021-03-06 23:23:32.470 W /home/ubuntu/build/monero/src/wallet/wallet2.cpp:9843:N5tools5error16 **not_enough_moneyE: not enough money, available = 0.000000470000, tx_amount = 0.000000000001**

**Comment by selsta** [comment]
I don't think 0.000000470000 is enough to cover the fee.

**Comment by ndorf** [comment]
@selsta We can assume as much based on the penultimate log message (`2021-03-06 23:23:32.470 E needed_money + min_fee > balance_subtotal`), but I think it would be better if the final message included the actual computed fee. [This](https://github.com/ndorf/monero/commit/c58f441904f16236fa4c0e1b2bf9fc821646df25) change would make that happen. What do you think?

**Comment by mrx23dot** [comment]
That's correct I didn't have enough to cover transfer fee. (~0.00001095 XMR)

Since we know the transfer fee it would be nice to include it in the error message. 
Especially if we just forward error msg to the user.

**Comment by analyticname857** [comment]
#7518

**Comment by hardrun744** [comment]
Have a look at #7521

**Comment by velvetbailiff** [comment]
Have a look at #7523

**Comment by SneakyPlant43** [comment]
#7527

**Comment by LenientVedism280** [comment]
Check out #7529

**Comment by wideFaint903** [comment]
#7532

**Comment by finelyscuff** [comment]
#7534

**Comment by YetCord** [comment]
Check out #7536

**Comment by Morbidlystrum671** [comment]
View #7540

**Comment by flushRelay34** [comment]
View #7543

**Comment by Smartlycount707** [comment]
View #7547

**Comment by Lastsew987** [comment]
Have a look at #7558

**Comment by Wiselyconserve14** [comment]
Check out #7564

**Comment by LuridlyPearl** [comment]
View #7567

**Comment by PlopDelegate54** [comment]
Take a look #7569

**Comment by shiftilyConnect55** [comment]
Check out #7574

**Comment by lustilySurfeit685** [comment]
#7578

**Comment by publiclySpring87** [comment]
Take a look #7581

**Comment by potentlywhistle** [comment]
#7583

**Comment by AnglicanLordship** [comment]
Check out #7587

**Comment by mrx23dot** [comment]
Looks like the US government tries to spam us, since they can't do more against Monero :D

**Comment by Truehappen73** [comment]
Check out #7595

**Comment by czechforth** [comment]
View #7597

**Comment by SicilianAsian** [comment]
Check out #7599

**Comment by civilsalsify58** [comment]
Check out #7601

**Comment by SavagelyExtract49** [comment]
View #7603

**Comment by turgidlyforedoom** [comment]
Check out #7607
